src/olympia/amo/cron.py

```diff
@@ -92,6 +95,18 @@ def days_ago(days):
         for line in output.split("\n"):
             log.debug(line)
 
+    # Delete stale FileUploads.
+    for file_upload in FileUpload.objects.filter(
+            created__lte=days_ago(180)).order_by('id'):
+        log.debug(u'[FileUpload:{uuid}] Removing file: {path}'
+                  .format(uuid=file_upload.uuid, path=file_upload.path))
+        if file_upload.path:
+            try:
+                storage.delete(file_upload.path)
+            except OSError:
+                pass
```

Comment:
We might want to log that exception, don't we? I don't think we expect that case to happen, or is there a reason why we might?